### PR TITLE
watch no longer builds styleguide

### PR DESCRIPTION
**Pull request by antialias**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "build-examples": "npm run build && webpack --context examples --config examples/webpack.config.js",
     "build-test": "webpack --context test --config test/webpack.config.js",
     "deploy-design": "./deploy-design.sh",
-    "watch": "npm run build && concurrently 'node build.js -w' 'webpack --context examples --config examples/webpack.config.js -w'",
+    "watch-styleguide": "npm run build && concurrently 'node build.js -w' 'webpack --context examples --config examples/webpack.config.js -w'",
```
Comment (tdumitrescu) [inline package.json]: TBH i don't think anyone uses this; we might as well delete it, now that running `build-examples` is basically preparation for publishing the style guide.

File: package.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
...
+    "watch": "npm run build && concurrently 'node build.js -w'",
```
Comment (tdumitrescu) [inline package.json]: `concurrently` a single thing is just the single thing, no?

Comment (antialias) [inline package.json]: never played with `concurrently` myself; I was just going with what was already there.

File: package.json (hunk 1 above)
Comment (antialias) [inline package.json]: ok

**Comment by antialias** [comment]
bump @tdumitrescu

File: package.json (hunk 2 above)
Comment (tdumitrescu) [inline package.json]: just get rid of that superfluous `concurrently` call in the watch task and it'll be good to go

**Pull request merged**